.github/workflows/stack-linter.yml

```diff
@@ -42,7 +42,7 @@ jobs:
       # Run Linter against code base #
       ################################
       - name: Lint Code Base
-        uses: docker://github/super-linter:latest
+        uses: github/super-linter:latest
```

Comment:
This new syntax is causing the workflow file to fail. Check out https://github.com/github/rally/pull/99/checks

Reply:
fixed the syntax to use v3 for now
since the update to actions, you don't need to specify docker, but you need to follow better at the end :)